src/TConsole.cpp

```diff
@@ -970,18 +972,12 @@ void TConsole::toggleLogging(bool isMessageEnabled)
             QString log;
             QTextStream logStream(&log);
             /*
-             * From the Qt Documentation:
-             * 'On Windows, the codec will be based on a system locale. On Unix
-             * systems, the codec will might fall back to using the iconv
-             * library if no builtin codec for the locale can be found."
-             *
-             * Note that in these cases the codec's name will be "System".'
-             *
-             * So if we are going to use UTF-8 as we declare in the HTML
-             * header we had better set that codec to be used:
+             * Setting a QTextCodec on a QString based QTextStream is
+             * ineffective as the Qt documents note: "When QTextStream operates
+             * on a QString directly, the codec is disabled."!
+             * Instead we must set it on the QTextStream that eventually
+             * receives the text - in this case mLogStream.
```

Comment:
This comment is not helpful. When you see the new code, the comment only explains why a previous code was discontinued. It also mentions a necessary change some 30 lines earlier. Instead, the comment should not mention the discontinued code at all. That is what git and github are used for. The comment should be 30 lines earlier. It should also explain, why a codec for locale is used. Much like the comment did until this change.

Reply:
Okay - will pare this down as far as possible - to a one-liner if I can - but it is still going to be worth mentioning here why we are NOT using a codec HERE. Also will look to put a briefer explanation where it IS done.